e2etest/pkg/frr/bgp.go

```diff
@@ -0,0 +1,40 @@
+package frr
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"go.universe.tf/metallb/e2etest/pkg/container"
+)
+
+// TODO: Leaving this package "test unaware" on purpouse, since we may find it
+// useful for fetching informations from FRR (such as metrics) and we may need to move it
+// to metallb.
+
+// NeighborForContainer returns informations for the given neighbor in the given
+// container. If containerName is "", it queries localhost.
+func NeighborForContainer(neighborName, containerName string) (*Neighbor, error) {
+	res, err := container.RunCommand(containerName, "vtysh", "-c", fmt.Sprintf("show bgp neighbor %s", neighborName))
```

Comment:
why not use 'show bgp neighbor %s json' and parse that instead of the regexes?

Reply:
because I was not aware of such option! Having a machine friendly format is definetely better, will change the parsing mechanism.

Reply:
Also, if this may be general, it might be good to start creating public(?) functions in order to hide how we connect to frr. Something like: FrrShowBGPNeighbor()

Also, it would be OK to implement this at a later stage.